### PR TITLE
Make cleanup with constants

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Make cleanup with constants.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/util/constants.go [hunk 1]
```diff
@@ -158,19 +161,17 @@ const (
 	HorizontalPodAutoscalerKind = "HorizontalPodAutoscaler"
 	// PodDisruptionBudgetKind indicates the target resource is a poddisruptionbudget
 	PodDisruptionBudgetKind = "PodDisruptionBudget"
+	// ClusterRoleKind indicates the target resource is a clusterrole
+	ClusterRoleKind = "ClusterRole"
+	// ClusterRoleBindingKind indicates the target resource is a clusterrolebinding
+	ClusterRoleBindingKind = "ClusterRoleBinding"
+	// CRDKind indicated the target resource is a CustomResourceDefinition
```
Comment (whitewindmills) [inline pkg/util/constants.go:168]: `indicates`

Comment (XiShanYongYe-Chang) [reply pkg/util/constants.go]: Thanks, updated.

**Review by RainbowMango — COMMENTED** [review]
/lgtm
/assign @whitewindmills 
for final decision.

**Pull request merged**